ci/raydepsets/cli.py

```
@@ -92,11 +92,20 @@ def execute(self):
             depset = self.build_graph.nodes[node]["depset"]
             self.execute_single(depset)
 
-    def get_depset(self, name: str) -> Depset:
+    def get_depset(self, name: str, build_arg_set: Optional[str] = None) -> Depset:
         for depset in self.config.depsets:
-            if depset.name == name:
+            if depset.name != name:
+                continue
+
+            if build_arg_set:
+                if depset.build_arg_set and depset.build_arg_set.name == build_arg_set:
+                    return depset
+            else:
```

maybe have a separate function that looks up the depset based on name?

why does `depset.build_arg_set` has a name?

and could you raise an explicit error when the depset has a `build_arg_set` member but `build_arg_set` argument is `None`? currently it will just return the depset.

or maybe just that I am confused or missing something?